modules/oe_whitelabel_search/src/Form/SearchForm.php

```diff
@@ -45,58 +45,33 @@ public static function create(ContainerInterface $container) {
    * {@inheritdoc}
    */
   public function buildForm(array $form, FormStateInterface $form_state, array $config = NULL): array {
+    if (empty($config['input']['name'])) {
+      return [];
+    }
```

Comment:
Why do we add a check for this one value, if all the other values are assumed as given?
Is there a reason why this one value might be empty, but the others not?

Reply:
Whatever, keep it.